apps/styleguide/src/tables/TableExplorer.vue

```diff
@@ -179,29 +189,29 @@
 </template>
 
 <script>
-import TableMolgenis from "./TableMolgenis";
-import FilterSidebar from "./FilterSidebar";
-import FilterWells from "./FilterWells";
-import MessageError from "../forms/MessageError";
-import RowButtonAdd from "./RowButtonAdd";
-import RowButtonDelete from "./RowButtonDelete";
-import RowButtonEdit from "./RowButtonEdit";
-import RowButtonClone from "./RowButtonClone";
-import Spinner from "../layout/Spinner";
-import TableMixin from "../mixins/TableMixin";
-import ShowHide from "./ShowHide";
-import InputSearch from "../forms/InputSearch";
-import Pagination from "./Pagination";
-import ButtonAlt from "../forms/ButtonAlt";
-import SelectionBox from "./SelectionBox";
-import ButtonDropdown from "../forms/ButtonDropdown";
-import InputSelect from "../forms/InputSelect";
-import TableCards from "./TableCards";
-import IconAction from "../forms/IconAction";
-import RecordCard from "./RecordCard";
-import TableSettings from "./TableSettings";
-
-const View = { TABLE: "table", CARDS: "cards", RECORD: "record", EDIT: "edit" };
+import TableMolgenis from './TableMolgenis';
+import FilterSidebar from './FilterSidebar';
+import FilterWells from './FilterWells';
+import MessageError from '../forms/MessageError';
+import RowButtonAdd from './RowButtonAdd';
+import RowButtonDelete from './RowButtonDelete';
+import RowButtonEdit from './RowButtonEdit';
+import RowButtonClone from './RowButtonClone';
+import Spinner from '../layout/Spinner';
+import TableMixin from '../mixins/TableMixin';
+import ShowHide from './ShowHide';
+import InputSearch from '../forms/InputSearch';
+import Pagination from './Pagination';
+import ButtonAlt from '../forms/ButtonAlt';
+import SelectionBox from './SelectionBox';
+import ButtonDropdown from '../forms/ButtonDropdown';
+import InputSelect from '../forms/InputSelect';
+import TableCards from './TableCards';
+import IconAction from '../forms/IconAction';
+import RecordCard from './RecordCard';
+import TableSettings from './TableSettings';
+import Panel from '../layout/Panel';
+const View = {TABLE: 'table', CARDS: 'cards', RECORD: 'record', EDIT: 'edit'};
```

Comment:
keeping an enter after the imports would be nice :)

Reply:
Yes, changed